scripts/install_environment.sh

```diff
@@ -43,6 +43,22 @@ function install_libvirt() {
         swtpm \
         swtpm-tools
```

Comment:
@nmagnezi BTW
with those still in place, are you sure the logic of the fix is actually being used?
because the binaries might still use dynamically linked libraries which are of the original code

Reply:
I left it in place because I tried to avoid deploying dependencies manually.but sure we can try to remove and see what happens.